deploy/ray-operator/controllers/common/pod.go

```diff
@@ -0,0 +1,132 @@
+package common
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	rayiov1alpha1 "ray-operator/api/v1alpha1"
+	"strconv"
+	"strings"
+)
+
+type PodConfig struct {
+	RayCluster  *rayiov1alpha1.RayCluster
+	PodTypeName string
+	PodName     string
+	Extension   rayiov1alpha1.Extension
+}
+
+func DefaultPodConfig(instance *rayiov1alpha1.RayCluster, podTypeName string, podName string) *PodConfig {
+	return &PodConfig{
+		RayCluster:  instance,
+		PodTypeName: podTypeName,
+		PodName:     podName,
+	}
+}
+
+// The function buildPod build pod for cluster instance.
+func BuildPod(conf *PodConfig) *corev1.Pod {
+	// build label for cluster
+	rayLabels := labelsForCluster(*conf.RayCluster, conf.PodName, conf.Extension.Labels)
+	// add pod type to label
+	rayLabels[ClusterPodType] = conf.PodTypeName
```

Comment:
maybe it would be better to wrap the pod type in func `labelsForCluster`.

Reply:
sure